extension/intents/scroll/scrollHelper.js

```diff
@@ -2,8 +2,28 @@
 
 const scrollAmount = 0.9;
 
-function scrollVertically(dy, smooth) {
-  window.scrollBy({
+function getScrollableElement() {
+  let element = document.activeElement;
+  let height = element.offsetHeight;
+  let scrollPos = element.scrollTop;
+  const scrollHeight = element.scrollHeight;
+
+  if (document.contentType === "text/html") {
+    element = window;
+    height = element.innerHeight;
+    scrollPos = element.scrollY;
+  }
+
+  return {
+    element,
+    height,
+    scrollPos,
+    scrollHeight,
+  };
+}
```

Comment:
While this works for the PDF example, it won't work for other pages, like a Bugzilla bug (https://bugzilla.mozilla.org/show_bug.cgi?id=343205) or a Google doc. I think the `document.contentType` check isn't the right thing: rather `document.activeElement` _or one of its ancestors_ is always the right thing to scroll.

This comment also gives some guidance: https://github.com/mozilla/firefox-voice/issues/954#issuecomment-591691095